corehq/apps/locations/fixtures.py

```diff
@@ -250,20 +246,15 @@ def _get_include_without_expanding_locations(domain, location_type):
     """
     include_without_expanding = location_type.include_without_expanding
     if include_without_expanding is not None:
-        forced_location_level = set(
-            SQLLocation.active_objects.
-            filter(domain__exact=domain, location_type=include_without_expanding).
-            values_list('level', flat=True)
-        ) or None
+        forced_location_level = (SQLLocation.active_objects
+                                 .filter(domain__exact=domain, location_type=include_without_expanding)
+                                 .values_list('level', flat=True)
+                                 .first())
```

Comment:
Similar to an earlier change - just use the first matching location to get the level, no need to look at all of them.